Sprint-1/errors/3.js

```diff
@@ -1,4 +1,5 @@
 const cardNumber = 4533787178994213;
+cardNumber = cardNumber.toString();
```

Comment:
Constants cannot be reassigned. There must be a different way to go about this.

Reply:
I believe the toString content can be stored in a new variable. Or I could use the let declaration instead of the const7